### ISSUE TITLE
Uploaded objects with zero payload instead of expected data

### ISSUE THREAD

**Issue by anatoly-bogatyrev** [nspcc-dev/neofs-http-gw#122]
When loading objects via NeoFS.send, periodically (not every time, which complicates reproducibility), an object with a zero payload is created.

Head example:
```
$ neofs-cli ... object head --cid 4Pm2ptHQQ3pq6euM9PnEP3MgPC65NRPmFJd3VPhYqzTi --oid 2koFXqVTV6xHhBJ4hHW2mvs12wmoS19NXSL9vfRXw97x 
ID: 2koFXqVTV6xHhBJ4hHW2mvs12wmoS19NXSL9vfRXw97x
CID: 4Pm2ptHQQ3pq6euM9PnEP3MgPC65NRPmFJd3VPhYqzTi
Owner: NceCu75mcC9pCu9D8ykj1jyZQGjzXcitGZ
CreatedAt: 3088
Size: 0
HomoHash: 00000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001
Checksum: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
Type: regular
Attributes:
  Owner-Email=anatoly@nspcc.ru
  Email=7a7ba8cf152b04e2c39218f4ee331917f9e935113e65ada6304015fe88a24dd8
  __NEOFS__EXPIRATION_EPOCH=3090
  FileName=photo_2021-01-25_12-12-38.jpg
Split Header Signature:
  public key: 031445f5788a237355912f71ec80a7e317250a8c099542a2049ddcf96dbbb33b8d
  signature: 041891b73770ce153826361d0a4d0fd3324ba1ebcdc22cbd8f4b4299d2dc3aaf6ee847365708dad7d029eb6cdb2fc2717f21787277ae93ebbf977a92cf1a142e47
(prod:neofs)
```

**Comment by KirillovDenis** [comment]
Are there some debug logs in http-gw? Something like `ignore part, empty form name` or `ignore part, empty filename`?

**Comment by KirillovDenis** [comment]
Some time ago I can reproduce this issue on NeoFS.send every 2-3 uploading. But now I cannot (http-gw version was changed but I doubt it's related). Also I couldn't reproduce this bug using local environment.

It would be nice to have `http-gw` logs for further investigation.
/cc @anatoly-bogatyrev

**Comment by alexvanin** [comment]
@KirillovDenis As soon as stage was updated with neofs-http-gw v0.18.0, can you reproduce it every 2-3 uploading?

**Comment by KirillovDenis** [comment]
Blocked by https://github.com/nspcc-dev/neofs-sdk-go/issues/126
